src/backend/launcher.ts

```diff
@@ -537,6 +537,7 @@ async function runWineCommand({
 
   return new Promise<{ stderr: string; stdout: string }>((res) => {
     const wrappers = options?.wrappers || []
+    const { launcherArgs = '' } = settings
```

Comment:
This should be done in the launch function for sideloaded games instead. You don't want these options all the time

Reply:
good point, just fixed it.